xlive/Blam/Engine/math/real_math.h

```diff
@@ -2,6 +2,11 @@
 
 /* constants */
 
+#define _REAL_MATH_USE_LEGACY_FPU_INSTRUCTIONS 1
+
+// faster than actual calls to trig functions
+#define _REAL_MATH_USE_LEGACY_TRIG_INSTRUCTIONS 1
```

Comment:
Please remove these and the assembly as we discussed prior.
Leave the helper functions so they just use the floating point versions of the global clib math functions for now...

Reply:
No, these will remain as they are, FPU instructions respect the precision configuration set using _controlfp/_control87,  which is configured by `real_math_initialize()` to _PC_54 (i.e double)